spk/go2rtc/Makefile

```diff
@@ -10,7 +10,7 @@ UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 MAINTAINER = skrashevich
 DESCRIPTION = Ultimate camera streaming application with support RTSP, RTMP, HTTP-FLV, WebRTC, MSE, HLS, MP4, MJPEG, HomeKit, FFmpeg, etc
 DISPLAY_NAME = go2rtc
-CHANGELOG = "Add expr source<br />Add support multiline exec source<br />Add IPv6 support to WebRTC <br />Add more ffmpeg transcoding presets<br />Add support host candidate config<br />Increase buffer for RTP producers<br />Update WebRTC candidates logic <br />Fix pixel format for h264 vaapi hardware transcoding<br />Fix ngrok support<br />Fix wrong SDP for MERCURY camera <br />Fix panic for HomeKit source without SRTP module <br />Fix raw conn for https client (wss transport for rtsp source)<br />Fix memory overflow on bad RTSP sources "
+CHANGELOG = "Add support ADTS to magic producer. Improve support incoming MJPEG in some cases. Fix https source in some cases. Fix api port for homekit module. Add new GoPro source for streaming from GoPro cameras  Add support unix socket for api module  Add support incoming H264 bitstream Fix https source   Fix support Tapo C210 firmware v1.3.9  Fix homekit source panic on stop producer Fix panic for broken RTP with AAC  Fix digest auth in some cases"
```

Comment:
For your changelog it is recommended that you separate discrete lines with numbers and `<br/>` line breaks as in the example shown in [Makefile variables](https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables#available-variables-in-spk-makefiles).